web/src/data/superrare/queries/getSuperrareLiveAuctionDetails.ts

```diff
@@ -87,13 +87,15 @@ export async function getSuperrareLiveAuctionDetails(utid: string): Promise<Live
 
   const startTime = auction?.startTime ?? 0;
   const endTime = auction?.endTime ?? 0;
-  const imageSrc = nft?.metadata?.originalMediaUri ?? nft?.metadata?.proxyImageMediumUri;
+  const imageSrc = nft?.metadata?.proxyImageMediumUri ?? nft?.metadata?.originalMediaUri;
```

Comment:
For video content, take image proxy if available.
Still need to add support for avif streams